### PR TITLE
Handle RV in Get calls to storage interface.

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #37473

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37692)

**Comment by wojtek-t** [comment]
@k8s-bot cvm test this, issue: #37168

**Comment by wojtek-t** [comment]
@k8s-bot etcd3 test this, issue: #37168

**Comment by wojtek-t** [comment]
@k8s-bot gci gce test this, issue: #37168

**Comment by wojtek-t** [comment]
Trivial rebase - readding label.

**Comment by wojtek-t** [comment]
@k8s-bot etcd3 test this, issue: #27673

File: pkg/storage/interfaces.go [hunk 1]
```diff
@@ -124,7 +124,9 @@ type Interface interface {
 	// Get unmarshals json found at key into objPtr. On a not found error, will either
 	// return a zero object of the requested type, or an error, depending on ignoreNotFound.
 	// Treats empty responses and nil response nodes exactly like a not found error.
-	Get(ctx context.Context, key string, objPtr runtime.Object, ignoreNotFound bool) error
+	// The returned contents may be delayed, but it is guaranteed that they will
+	// be have at least 'resourceVersion'.
```
Comment (hongchaodeng) [inline pkg/storage/interfaces.go]: Also note that if resourceVersion is empty, it will provide strong consistency guarantee.

**Comment by wojtek-t** [comment]
@k8s-bot node test this, issue: #IGNORE

**Pull request merged**